### ISSUE TITLE
Provide and option to hide the menu bar icon

### ISSUE THREAD

**Issue by brechtm** [dmarcotte/easy-move-resize#17]
I'm sure I'm not the only one who's menu bar is slowly filling up with icons. Easy Move+Resize's icon doesn't offer much value once it has been configured, so it would be great if it could be hidden. Starting the app if it's already running could open a configuration window (with a single checkbox to hide the menu bar icon).

**Comment by dmarcotte** [comment]
(Apologies for the late reply)  I understand where this idea comes from, but I'm not sure I want to add this capacity directly to the app.  I use [Bartender](https://www.macbartender.com/) to wrangle that slow-filling menu bar and recommend it, but I understand that answer probably won't be satisfying for everyone.

Will leave this open since I'd likely accept a pull if it was pretty straightforward, but currently my leaning is to not implement anything here.

**Comment by romant** [comment]
agreed; many of the bar items can be easily hidden, as also use Bartender

@brechtm - check out [vanilla](http://matthewpalmer.net/vanilla/) ( free ) if you want a Bartender alternative

**Comment by brechtm** [comment]
@romant I recently discovered Vanilla. Unfortunately, it simply hides the items by overlaying it with a grey rectangle, it seems. This doesn't solve the issue with applications that have many menus.

Ideally, Bartender's functionality should be part of macOS. It should no be necessary for a 3rd-party "hack" like Bartender. Since Apple doesn't seem to agree, it's nice to be able to configure applications to not display items in the menu bar.

**Comment by Rabattkarte** [comment]
As of macOS 26 (Tahoe) you can hide menu bar items in Settings --> Menu Bar. This would make this feature request a QoL feature for everyone on macOS < 26.
